model/facade/action/GetPersonalSummaryByLoginDateAction.php

```diff
@@ -198,7 +208,7 @@ protected function doExecute() {
                 $weeksInBetween = $this->getWeeksInBetweenDates( $this->currentUserGoal->getInitDate(), $this->currentUserGoal->getEndDate() );
                 $extraGoalHoursSet += ( $this->currentUserGoal->getExtraHours() / $weeksInBetween );
             }
-            $originalHoursToBeWorked = round(($this->getWorkableHoursInThisJourneyPeriod() - $this->getWorkedHoursInThisJourneyPeriod())/ $this->getWeeksTillEndOfJourneyPeriod() , 2);
+            $originalHoursToBeWorked = round(($this->getWorkableHoursInThisPeriod() - $this->getWorkedHoursInThisPeriod())/ $this->getWeeksTillEndOfJourneyPeriod() , 2);
             $totalResults['weekly_goal'] = floor( ( $originalHoursToBeWorked + $extraGoalHoursSet ) * 60);
```

Comment:
This calculation is wrong because `$originalHoursToBeWorked` and `$extraGoalHoursSet` are not comparable. Imagine a goal betweek Jan, 4 and Dec, 31, and the current date is Aug, 1:
```
weeksInBetween = 52
weeksTillEndOfJourneyPeriod = 22

$originalHoursToBeWorked = (workable - worked) / 22 weeks
$extraGoalHoursSet = (extra hours) / 52 weeks
```

The proper way to do this would be:
```
$totalResults['weekly_goal'] = (workable - worked + currentUserGoal) / weeksTillEndOfJourneyPeriod * 60